DuckDuckGo/TabViewController.swift

```diff
@@ -828,7 +833,15 @@ class TabViewController: UIViewController {
         Pixel.fire(pixel: .privacyDashboardOpened)
         performSegue(withIdentifier: "PrivacyDashboard", sender: self)
     }
-    
+
+    func setRefreshControlEnabled(_ isEnabled: Bool) {
+        if isEnabled {
+            webView.scrollView.refreshControl = refreshControl
+        } else {
+            webView.scrollView.refreshControl = nil
+        }
+    }
```

Comment:
lets make it 1-liner

webView.scrollView.refreshControl = isEnabled ? refreshControl : nil